tags/piracy.tag

```diff
@@ -3,7 +3,7 @@ type: text
 ---
 
 Cracked servers are not supported here and by setting your server as so, you will not receive support here any further.
-"Cracked" or "offline mode" servers basically disable any sort of Mojang authentication and allow people with non-genuine versions of the game to join. **This is software piracy and is illegal!**. By setting your server as so, you are condoning the use of piracy. 
+"Cracked" or "offline mode" servers basically disable any sort of Mojang authentication and allow people with non-genuine versions of the game to join. Unless the server is behind an online mode BungeeCord instance, **This is software piracy and is illegal!**. By setting your server as so, you are condoning the use of piracy. 
```

Comment:
Still, the server being in offline mode isn't piracy itself... I just don't like how harshly worded this whole thing is...

Reply:
I think the wording is fine how it is. By setting your server to offline mode, anyone can join the server without having bought the game to begin with. I don't see much of a valid usecase otherwise, unless someone has no internet, which in that case, they most likely would not be on our Discord at all when it comes to support. In the event that they're on another device or something, we can likely be more lenient with them.